instagram/src/App.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import dummyData from './dummy-data';
+import PostsPage from './components/PostContainer/PostsPage';
+import Authenticate from './components/Authentication/Authenticate';
+
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      searchText: '',
+      isSelected: false,
+    }
+  }
+
+  logOut = () => {
+    localStorage.removeItem('username');
+    window.location.reload();
+  }
+
+  clearSearchText = () => {
```

Comment:
You don't appear to be using this anywhere but you shouldn't need a separate method for this anyway.  On a setState in which you're using the input value you can set the string to empty as an additional expression.